### ISSUE TITLE
Cannot start docker4ssh server

### ISSUE THREAD

**Issue by IchBinLeoon** [ByteDream/docker4ssh#1]
When I run the command `docker4ssh start`, I get the following error:

```
Error: Near line 39 (last key parsed 'ssh'): expected a top-level item to end with a newline, comment, or EOF, but got '"' instead
Usage:
  docker4ssh start [flags]

Flags:
  -h, --help   help for start

Near line 39 (last key parsed 'ssh'): expected a top-level item to end with a newline, comment, or EOF, but got '"' instead
```

Installed via yay from the AUR on Arch Linux.

**Comment by ByteDream** [comment]
Probably an error in the Makefile. A random passphrase for the ssh key is [generated](https://github.com/ByteDream/docker4ssh/blob/a5890141064a89c15413e73dc5ff22a7b5e08098/Makefile#L20) there and it somehow messes up the `sed` command which [places](https://github.com/ByteDream/docker4ssh/blob/a5890141064a89c15413e73dc5ff22a7b5e08098/Makefile#L28) the passphrase inside of the docker4ssh.conf. Best workaround is to re-install it or generating a new ssh key (`ssh-keygen -t ed25519 -f /etc/docker4ssh/docker4ssh.key -b 4096`) and place the entered passphrase into `/etc/docker4ssh/docker4ssh.conf`
```
...
[ssh]
Port = 2222
Keyfile = "./docker4ssh.key"
Passphrase = "<your entered passphrase here>"
...
```

**Comment by IchBinLeoon** [comment]
Reinstalling does not work, at least for me, and generating a new ssh key isn't necessary. 

![image](https://user-images.githubusercontent.com/50367411/146686181-97c74eda-e9d8-41e1-896e-6cb41b800100.png)

I have manually corrected the line that `sed` messed up. Now it seems to work.

**Comment by IchBinLeoon** [comment]
Installed it again from the AUR, this time it works directly and the passphrase line is not messed up. Looks like some characters or character combinations of the generated passphrase corrupt the sed command and I was unlucky twice in a row.

**Comment by ByteDream** [comment]
Fixed in [v0.1.1](https://github.com/ByteDream/docker4ssh/releases/tag/v0.1.1)
